server/src/main/java/com/aws/greengrass/cli/CLIEventStreamAgent.java

```diff
@@ -178,6 +189,10 @@ public void persistLocalDeployment(Topics serviceConfig, Map<String, Object> dep
         Topics localDeployments = serviceConfig.lookupTopics(PERSISTENT_LOCAL_DEPLOYMENTS);
         String deploymentId = (String) deploymentDetails.get(DEPLOYMENT_ID_KEY_NAME);
         Topics localDeploymentDetails = localDeployments.lookupTopics(deploymentId);
+        if (localDeploymentDetails.find(LOCAL_DEPLOYMENT_CREATED_ON) != null) {
+            deploymentDetails.put(LOCAL_DEPLOYMENT_CREATED_ON,
+                    Coerce.toLong(localDeploymentDetails.find(LOCAL_DEPLOYMENT_CREATED_ON)));
+        }
```

Comment:
Why do you need this? I thought your other changes on line 511 already made sure that `deploymentDetails` contains createdOn

Reply:
Yes, unfortunately persistLocalDeployment replaces and re populates the entire local deployment node with the newest update and the original `createdOn` gets lost. So some mechanism was required to persist createdOn.
https://github.com/aws-greengrass/aws-greengrass-cli/blob/tune-status/server/src/main/java/com/aws/greengrass/cli/CLIEventStreamAgent.java#L196

Reply:
ok i see